includes/definitions.inc.php

```diff
@@ -1363,7 +1363,7 @@
 $config['os'][$os]['icon'] = 'hikvision';
 
 // Canopy / Cambium support
-$os = 'canopy';
```

Comment:
Noted, in that sense. It would be divided into 2 OS's if this is okay.
Canopy OS:
Cambium fsk,450 SM/AP

Cambium OS:
PTP650
PTP250
ePTP

Those devices under the cambium don't have any OS that i can really label them besides cambium. Would this be ok?